### PR TITLE
Complete order list

### PR DESCRIPTION
**Pull request by ney601**

### CODE REVIEW

**Review by Jean-005 — APPROVED** [review]
hi5!

**Review by Tommybui28 — APPROVED** [review]
YES YOU CAN

**Review by sahilpatel1906 — APPROVED** [review]

File: capstone_frontend/src/components/lists/OrderList.jsx
Comment (Aebel-Shajan) [inline capstone_frontend/src/components/lists/OrderList.jsx:1]: Nice work team lets gooo big progress big gains

**Review by Aebel-Shajan — APPROVED** [review]

File: capstone_frontend/src/components/lists/OrderList.jsx
Comment (Aebel-Shajan) [reply capstone_frontend/src/components/lists/OrderList.jsx:1]: apologies for the wait 🥺

**Pull request merged**